crates/devolutions-pedm/openapi/generate_all.ps1

```diff
@@ -0,0 +1,19 @@
+#!/bin/env pwsh
+
+$ErrorActionPreference = "Stop"
+
+Push-Location -Path $PSScriptRoot
+
+try {
+	../../../tools/generate-openapi/generate.ps1 pedm | Out-Host
```

Comment:
I believe its better to write path relative to script root ($PSScriptRoot variable) isntead of relative path (`../../..`)
cc @awakecoding

Reply:
It’s fine, because there is a `Push-Location -Path $PSScriptRoot` at the top of the file, ensuring we are always relative to the file.